includes/html/pages/device/health/processor.inc.php

```diff
@@ -1,26 +1,19 @@
 <?php
 
-$graph_type = 'processor_usage';
-
 $i = '1';
 
-if (count_mib_processors($device) > 0) {
-    $processors = get_mib_processors($device);
-    $graph_type = 'device_processor';
-} else {
-    $processors = dbFetchRows('SELECT * FROM `processors` WHERE device_id = ?', array($device['device_id']));
-}
+$processors = dbFetchRows('SELECT * FROM `processors` WHERE device_id = ?', array($device['device_id']));
 
 foreach ($processors as $proc) {
-    if ($graph_type == 'device_processor') {
+    if ('processor_usage' == 'device_processor') {
```

Comment:
stringa == stringb ?
will never match, or did i miss something?

Reply:
mass replace :D  because $graph_type is always processor_usage now.  I'll fix it, nice catch.

Reply:
was just a fly over :)
happens ...